### PR TITLE
Better meta tags

### PR DESCRIPTION
**Pull request by KMontag42**
these look pretty dank now

### CODE REVIEW

File: components/FormationCard.tsx [hunk 1]
```diff
@@ -1,10 +1,4 @@
-"use client";
```
Comment (SwiftieHusband) [inline components/FormationCard.tsx:1]: Wait does this default to server or client?

**Review by SwiftieHusband — DISMISSED** [review]

File: components/FormationCard.tsx [hunk 2]
```diff
@@ -1,10 +1,6 @@
...
+'use client'
```
Comment (SwiftieHusband) [inline components/FormationCard.tsx:1]: single quotes, no closing semi colon...

File: lib/cms-export.json
Comment (SwiftieHusband) [inline lib/cms-export.json:1]: Is the point of this file just to keep a hard copy?

Comment (KMontag42) [reply lib/cms-export.json:1]: yeah exactly

**Review by SwiftieHusband — APPROVED** [review]

File: components/FormationCard.tsx (hunk 1 above)
Comment (KMontag42) [reply components/FormationCard.tsx:1]: for now this will stay as a client component until i refactor the rendering some more

**Pull request merged**